libs/design/src/molecules/qty-dropdown/qty-dropdown.module.ts

```diff
@@ -3,11 +3,13 @@ import { CommonModule } from '@angular/common';
 import { FormsModule } from '@angular/forms';
 
 import { DaffQtyDropdownComponent } from './qty-dropdown.component';
+import { DaffSelectModule } from '../../atoms/form/select/select.module';
 
 @NgModule({
   imports: [
     CommonModule,
-    FormsModule
+    FormsModule,
+    DaffSelectModule
```

Comment:
DaffInputModule should be imported here too

Reply:
fixed in 220b26d